flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/pipeline/Predictor.scala

```diff
@@ -121,7 +121,7 @@ object Predictor {
 
         input.mapWithBcVariable(model){
           (element, model) => {
-            (element, predictOperation.predict(element, model))
+            (element, predictOperation.predict(element, model, resultingParameters))
```

Comment:
I could have a `predictParameters` variable in the SVM `PredictOperation` and assign it a value in `getModel` then.

Does that sound better?